code/game/objects/items/weapons/rpd.dm

```diff
@@ -195,6 +193,22 @@
 	QDEL_NULL(P)
 	activate_rpd()
 
+/**
+* Automatically wrenches down an atmos device/pipe if the auto_wrench_toggle is TRUE.
+* Arguments:
+* * user - the user of the RPD.
+* * target - the pipe/device/tube being placed by the RPD.
+*/
```

Comment:
Devdocs on multiline comments is [Here](https://devdocs.paradisestation.org/coding/style_guidelines/#multi-line-comments), tldr just add a space before every * !